vendor/google.golang.org/grpc/clientconn.go

```diff
@@ -42,10 +42,12 @@ import (
 	"google.golang.org/grpc/internal/transport"
 	"google.golang.org/grpc/keepalive"
 	"google.golang.org/grpc/resolver"
-	_ "google.golang.org/grpc/resolver/dns"         // To register dns resolver.
-	_ "google.golang.org/grpc/resolver/passthrough" // To register passthrough resolver.
 	"google.golang.org/grpc/serviceconfig"
 	"google.golang.org/grpc/status"
+
+	_ "google.golang.org/grpc/balancer/roundrobin"           // To register roundrobin.
+	_ "google.golang.org/grpc/internal/resolver/dns"         // To register dns resolver.
```

Comment:
@YoyinZyc similar to my above comment - while looking at the couple of issues https://github.com/etcd-io/etcd/issues/11532 and https://github.com/etcd-io/etcd/issues/11529
, I think that these issues are might be because of https://github.com/etcd-io/etcd/tree/master/vendor/google.golang.org/grpc/internal doesn't have the referenced `internal/resolver` package. The https://github.com/grpc/grpc-go/tree/master/internal has it there. I haven't worked on updating etcd vendor files before but I believe having vendor dir updated with these packages should fix it. Please let me know what you think? Thanks!